public/js/app/controllers/PostAttachmentController.js

```diff
@@ -1,6 +1,14 @@
 define(["app/app",
-        "ember"], function(App, Ember) {
+        "ember",
+        "numeral"], function(App, Ember, numeral) {
   "use strict";
 
-  App.PostAttachmentController = Ember.Controller.extend()
+  App.PostAttachmentController = Ember.Controller.extend({
+    tooltip: function() {
+      var fileName = this.get('model.fileName')
+      var fileSize = this.get('model.fileSize')
+      fileSize = numeral(fileSize).format('0.[0] b')
+      return fileName + ' (' + fileSize + ')'
+    }.property('tooltip')
```

Comment:
It should be a property of two attributes `property('model.fileName', 'model.fileSize')`. Actually, since fileName and fileSize are (sort of) static and could be even just `property()`